docs/modules/components/image/ts/views/examples/source.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { Image } from "pragmate-ui/image";
+
+export function Sources() {
+    const sources = [
+        {
+            media: "(max-width: 1280px)",
+            srcSet: "https://th.bing.com/th/id/OIP.EvzGc57hkNpeXisvI_7rwgHaE8?w=276&h=185&c=7&r=0&o=5&pid=1.7"
```

Comment:
I think could be better to set the sources item object with properties "min" and "max" instead of media and build the media structure inside the component. It will generate a better dx. @FranciscoSanvicente